tests/unit_tests/blas/level1/asum.cpp

```diff
@@ -23,17 +23,16 @@
 #include <limits>
 #include <vector>
 
+#include <gtest/gtest.h>
```

Comment:
When you apply clang-format, does it move this header file here? When i tried it, it was putting it below other ones.

Reply:
I'm not sure.. do you want me to move it below others?

Reply:
I'm not sure.. do you want me to move it below others?

Reply:
I think, it is fine as long as clang-format is happy about it.